docs/log-ingestion/loki.md

```diff
@@ -1,11 +1,30 @@
 # Loki
 
-### Download:
-- curl -O -L "https://github.com/grafana/loki/releases/download/v2.8.2/promtail-darwin-amd64.zip"
-- unzip promtail-darwin-amd64.zip
+Loki is a log aggregation system designed to store and query logs from all your applications and infrastructure.
```

Comment:
add  below with some better wording...
`You can effortlessly migrate from your existing Loki installation to SigLens. SigLens supports ingesting logs in loki format and it is an easy drop-in replacement for Loki`